### DISCUSSION TITLE
Error: webgl finger print has block the track

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2108] [Q&A] [answered]
One of our user is getting the error below.  This happens after login and before loading iModel spinner shows up.  Nothing shows up in the viewport.  iTwin app was working properly on his machine until recently, but now he consistently getting this message and is unable to do anything.

![Tony-Error](https://user-images.githubusercontent.com/83007711/130264854-a973908f-e1b5-43c3-97dc-06b0f7638c24.JPG)

**Comment by aruniverse** [comment]
Have your user check their machine's compatibility with: https://connect-imodeljscompatibility.bentley.com/, and report back their results.

**Comment by pmconne** [comment, marked as answer]
The user is presumably using some kind of browser extension that produces that error message, such as [WebGL Fingerprint Defender](https://chrome.google.com/webstore/detail/webgl-fingerprint-defende/olnbjpaejebpnokblkepbphhembdicik?hl=en).
If so, the compatibility checker linked by @aruniverse will also fail.
Tell them to disable the extension for your app.
